### PR TITLE
feat: add doc for matrix

### PR DESCRIPTION
**Pull request by B3nnyL**

### CODE REVIEW

File: website/docs/docs/user-interface/dashboard.md [hunk 1]
```diff
@@ -26,9 +26,13 @@ If you have been assigned to a [team](#settings-teams), you can select them here
...
-The tabs at the top provide different views of the currently selected integration. The Network
-Diagram will display a graph view of all the integrations that are reachable from the selected one.
-The Matrix will display more details of all the pacts and verifications of the selected integration.
+The tabs at the top provide different views of the currently selected integration. 
+
+- The Network Diagram will display a graph view of all the integrations that are reachable from the selected one.
+
+- The [Matrix](#matrix) will display more details of all the pacts and verifications of the selected integration.
+
+- The Webhooks will display a list of triggered webhooks for the integration and logs
```
Comment (bethesque) [inline website/docs/docs/user-interface/dashboard.md:35]: Missing `.` on end.

File: website/docs/docs/user-interface/dashboard.md [hunk 2]
```diff
@@ -96,3 +100,19 @@ Clicking on a tag pill for a provider version tag will make the following menu a
 
 - Copy tag URL - selecting this menu item will copy the URL of the application version tag into the clipboard.
 - Copy tag name - selecting this menu item will copy the name of the tag into the clipboard.
+
+## Matrix 
+
+"Matrix" displays pacts and verifications of the selected integration.
```
Comment (bethesque) [inline website/docs/docs/user-interface/dashboard.md:106]: How about:

The "Matrix" displays a table that shows the verification status of each combination of consumer and provider version from the selected integration.

File: website/docs/docs/user-interface/dashboard.md [hunk 3]
```diff
@@ -96,3 +100,19 @@ Clicking on a tag pill for a provider version tag will make the following menu a
...
+
+&nbsp;
+
+![Matrix Screen](/ui/matrix.png)
+
+&nbsp;
+
+### Matrix Filters
+Filter your pacts and verifications of selected integration with consumer and provider's version, branch, environment and tag.
```
Comment (bethesque) [inline website/docs/docs/user-interface/dashboard.md:115]: "Filter the consumer versions and provider versions of the selected integration by version number, branch, environment or tag".

**Review by bethesque — CHANGES_REQUESTED** [review]

**Comment by bethesque** [comment]
Awesome, thanks!

File: website/docs/docs/user-interface/dashboard.md (hunk 2 above)
Comment (B3nnyL) [reply website/docs/docs/user-interface/dashboard.md:106]: It's way accurate, thanks!

**Pull request merged**